specifyweb/frontend/js_src/lib/components/Formatters/Fields.tsx

```diff
@@ -38,11 +38,11 @@ export function Fields({
            *   table layout with list layout
            */
           className={`
-            grid-table min-w-[35rem]
-            gap-y-4 gap-x-4
+            grid-table min-w-[35rem] gap-y-4 gap-x-4
+            [&_td]:items-stretch
```

Comment:
I didn't like that you had to add `h-full` in so many different places

reasoning:
`h-full` was needed because of the `grid-table` styling we have (`items-center`)
and given that `grid-table` is the cause of the issue, ideally, the solution for this would be somewhere in the grid table
where as by adding `h-full` in all those places, the cause of the problem and the solution to the problem are further away, making it less obvious what is the solution for

pushed a fix in https://github.com/specify/specify7/pull/4397/commits/3402921830119d428802b74b0d8d9a38305efa53